### DISCUSSION TITLE
With Github discussions now activated, can we use the Github Milestones as a roadmap?

### DISCUSSION THREAD

**Discussion by MysteriousOrb2000** [Docile-Alligator/Infinity-For-Reddit#886] [General]
Since Discussions is now used as a voting poll for feature requests, we could use the milestones as a roadmap for what's to come next like how other FOSS projects use it.

https://github.com/Docile-Alligator/Infinity-For-Reddit/milestones

**Comment by Wladefant** [comment]
Yes, of course, that is the next step we will make. We want as much transparency and community support as possible and are making changes according to that.
